artworks/templates/artworks/art_view.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class='art-view'>
+    <div class='img-box'>
+        <img class='img' src="/media/{{ artwork.image }}" alt="{{ artwork.title }} by {{ artwork.uploader.username }}"/>
+    </div>
+    <div class='info'>
+        <div class='art'>
+            <h1 class='color-text'>{{ artwork }}</h1>
+            <p class='white-text'>by</p>
+
+        </div>
+        <div class='artist'>
+            <img class='small-thumbnail' src="/{{ artist.profile.avatar }}" alt="{{ artist }}'s avatar"/>
+            <h2>{{ artist }}</h2>
+            <p class='white-text'>Followers</p>
+            <p class='color-text'>{{ artist.followers.count }}</p>
+        </div>
+        <div class='likes'>
+            {% if not liked %}
```

Comment:
Try to get used in doing "true" checks first. As in do `if liked ... else` instead of `if not liked ... else`. I know codewise it's the same, it's just easier to read and cleaner.